pkg/caveats/types/ipaddress.go

```diff
@@ -67,7 +71,7 @@ func (ipa IPAddress) Value() interface{} {
 	return ipa
 }
 
-var IPAddressType = registerCustomType(
+var IPAddressType = registerCustomType[IPAddress](
```

Comment:
the generic type here could be dropped too

Reply:
See above